### DISCUSSION TITLE
Silent install

### DISCUSSION THREAD

**Discussion by marie-bnl** [Syncplay/syncplay#446] [Ideas]
Hey! I'd like to make a Chocolatey packages. I know there's already an issue, the #358, but it didn't give anything.

So I don't want to talk about chocolatey or anything, I'd just like to know if you plan to add a silent switch for installing, or supporting the NSIS' ones, /S and /SD

Thanks!

**Comment by Et0h** [comment]
Seems like a good idea. I've never used the NSIS /S switch, but it should hopefully be relatively straightforward for someone to add support. I would welcome a pull request for this.

**Comment by Et0h** [comment]
For the avoidance of doubt, my focus for the foreseeable future is on fixing bugs, responding to issues, and maintaining the website and I don't expect to have time to take on any new responsibilities. As such, you or someone else would need to volunteer to implement this feature for this request to be implemented. I'll keep this issue open for a while in the hope that someone will make an appropriate pull request.

**Comment by marie-bnl** [comment]
Ok thanks! I'll see if I'm able to do a pull request then, but I don't think so, so I let others do that if someone wants
